### ISSUE TITLE
Terminal Email Client - Mutt

### ISSUE THREAD

**Issue by yashLadha** [iiitv/ShareIt#26]
Recently while watching videos on YouTube, i founded this amazing terminal email client which is pretty fast (cached 7000 mails in a sec) and also time saving, if you have a terminal workflow. It also enables us to send email with attachments straight from the terminal. Very handy tool. [Link](http://www.mutt.org/)

**Comment by harishkrupo** [comment]
True. The only drawback I found with this is that, if you are using Gmail-specific features like automatic category recognition, then I couldn't find out a way to get that working in mutt. This is important to me because Gmail automatically removes all the clutter (promotional emails, social feeds) from my mailbox, puts them into separate tabs and shows me only important messages. 
If this is not the case then mutt is extremely useful. Good that you shared :+1:

**Comment by yashLadha** [comment]
Sir, we can create multiple mailboxes in mutt, for different tabs. As explained in this [thread](https://productforums.google.com/forum/#!topic/gmail/xdnO4bHV-Cs). It seems that this is a flaw in mutt. Thanks for your response @harishkrupo
